lib/emque/producing/message/message_with_changeset.rb

```diff
@@ -0,0 +1,60 @@
+module Emque
+  module Producing
+    module MessageWithChangeset
+      include Emque::Producing::Message
+
+      def self.included(base)
```

Comment:
thinking about it, that does make sense. The attributes wouldn't be placed on the instance but instead on this module and then the including class.